xclim/sdba/utils.py

```diff
@@ -239,7 +239,7 @@ def broadcast(
 def equally_spaced_nodes(n: int, eps: Optional[float] = None) -> np.array:
     """Return nodes with `n` equally spaced points within [0, 1], optionally adding two end-points.
 
-    WARNING: Passing a small `eps` will effectively clip the scenario to the bounds of the reference
+    .. warning:: Passing a small `eps` will effectively clip the scenario to the bounds of the reference
```

Comment:
Why not use the:
```
Warnings
--------
```
RsT convention?

Reply:
I thought the "Warnings" section, as you suggest, was for listing what python warnings could be raised and why, while this rst block will show up in the main docstring as a not to the user.

After reading https://numpydoc.readthedocs.io/en/latest/format.html#warnings, I see that I am wrong.